<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapCoordsDialogBase</class>
 <widget class="QDialog" name="MapCoordsDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter map coordinates</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Enter X and Y coordinates which correspond with the selected point on the image. Alternatively, click the button with icon of a pencil and then click a corresponding point on map canvas of QGIS to fill in coordinates of that point.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>X</string>
       </property>
       <property name="buddy">
        <cstring>leXCoord</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="leXCoord"/>
     </item>
     <item row="0" column="2" rowspan="2">
      <widget class="QPushButton" name="btnPointFromCanvas">
       <property name="text">
        <string> from map canvas</string>
       </property>
       <property name="icon">
        <iconset resource="georeferencer.qrc">
         <normaloff>:/pencil.png</normaloff>:/pencil.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="leYCoord"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel2">
       <property name="text">
        <string>Y</string>
       </property>
       <property name="buddy">
        <cstring>leYCoord</cstring>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leXCoord</tabstop>
  <tabstop>leYCoord</tabstop>
  <tabstop>btnPointFromCanvas</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="georeferencer.qrc"/>
 </resources>
 <connections/>
</ui>
